crates/red_knot/tests/check.rs

```diff
@@ -1,6 +1,6 @@
-use red_knot::db::RootDatabase;
-use red_knot::lint::lint_semantic;
-use red_knot::workspace::WorkspaceMetadata;
```

Comment:
I think we could move this test to `red_knot_workspace`. It isn't CLI specific (arguably, it could even be in ruff python semantic).